game-of-life-native/Game of Life/SwiftGameLoop.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SwiftGameLoop.swift
+//  Game of Life
+//
+//  Created by Tobias Heine on 07.02.18.
+//  Copyright © 2018 Novoda. All rights reserved.
+//
+
+import UIKit
+import KotlinGameOfLife
+
+class SwiftGameLoop: NSObject,KGOLGameLoop {
+    
+    var displayLink : CADisplayLink!
```

Comment:
since it is not on `init` it would be better to have this as an optional to avoid possible crashes